### DISCUSSION TITLE
[closed][x][blueprint] Should we discern between tissue types and cell type etc?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#104] [Team Posts]
Seems there is structure for this in the BLUEPRINT data

**Comment by dzerbino** [comment]
OK, so sample description is hugely problematic so I would be weary of doing anything except superficial attributes and linking out to BioSamples. 
E.g.: 
- not all samples have tissue (e.g. yeast)
- not all samples have a cell type (e.g. unpurified tissue sample)

We could add optional attributes, but this list would certainly be incomplete, as different communities split their samples in different ways.

**Comment by sveinugu** [comment]
Could we add tissue_type as an optional field, and then filter out the tissue type info from the sample_type in the BLUEPRINT data?

**Comment by dzerbino** [comment]
Optional attributes are free, sure go ahead.

**Comment by sveinugu** [comment]
@dzerbino I have now added the term_iri and term_value subattributes of tissue_type also in the standard, so could you change the BLUEPRINT json accordingly?

**Comment by dzerbino** [comment]
Sure, I'll do it later today.
